common/changes/@uifabric/example-app-base/mapol-allow-overriding-root-element-on-link_2018-05-13-20-50.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
can you remove all of these change files that are type : none

Reply:
Yes can do and we should probably fix the tool :P

Reply:
Done!